outlier_detection/outlier_detect_ilf.py

```diff
@@ -0,0 +1,27 @@
+## Python verson 3.6
+## Replace the items in <> before running
+
+
+import pandas as pd
+data = pd.read_csv('<file_name>.csv', usecols=[<column_index, column_index, ...>]) # specifiy the dimensions to use in IsolationForest outlier detection
+
+
+# Use the following chunk if want to use log value of a column
+# ------------------------------------------------------------
+from math import log
+
+def cal_log(x):
+    if x == 0:
+        return 0
+    else:
+        return log(x)
+
+data['<column_name>'] = list(map(cal_log, data['<column_name>']))
```

Comment:
Move this into a block at the end of the file, which will be the main block of the program.  Rough organization of the program overall:

1. `import` statements
1. `def` of one or more functions
1. main program block